Asset/ProvisionedPathResolver.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the EzCoreExtraBundle package.
+ *
+ * (c) Jérôme Vieilledent <jerome@vieilledent.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Lolautruche\EzCoreExtraBundle\Asset;
+
+class ProvisionedPathResolver implements AssetPathResolverInterface
+{
+    /**
+     * @var array
+     */
+    private $resolvedPaths;
+
+    /**
+     * @var AssetPathResolverInterface
+     */
+    private $innerResolver;
+
+    public function __construct(array $resolvedPaths, AssetPathResolverInterface $innerResolver)
+    {
+        $this->resolvedPaths = $resolvedPaths;
+        $this->innerResolver = $innerResolver;
+    }
+
+    /**
+     * Looks for $path within pre-resolved paths for provided design.
+     * If it cannot be found, fallbacks to original resolver.
+     *
+     * {@inheritdoc}
```

Comment:
Here this is perfectly valid since I override the description, adding something, then I call the parent 😁